packages/commons-ui-core/src/Copyright/Copyright.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+
+const Copyright = React.forwardRef(function Copyright(props, ref) {
+  const {
+    copyright,
+    CopyrightProps,
+    icon,
+    IconProps,
+    variant,
+    year,
+    YearProps,
+    url,
+    sx,
+  } = props;
+
+  return (
+    <Box sx={sx} ref={ref}>
+      {copyright && (
+        <Typography variant={variant} {...CopyrightProps}>
+          {copyright}
+        </Typography>
+      )}
```

Comment:
1. Use `RichTypography` (not only because it can handle HTML, but also there is no need of doing the whole `copyright &&` check.
2. I'm assuming copyright and year use the same props; Wouldn't `TypographyProps` do do the job instead of two separate `CopyrightProps` and `YearProps` props?